src/windows.rs

```diff
@@ -0,0 +1,464 @@
+use crate::Error;
+use async_trait::async_trait;
+use futures::Stream;
+use maplit::hashmap;
+use std::collections::HashMap;
+use std::ffi::{c_void, OsString};
+use std::iter;
+use std::os::windows::prelude::*;
+
+use brightness_windows::Windows::Win32::{
+    Devices::Display::{
+        DestroyPhysicalMonitor, GetMonitorBrightness, GetNumberOfPhysicalMonitorsFromHMONITOR,
+        GetPhysicalMonitorsFromHMONITOR, SetMonitorBrightness, DISPLAYPOLICY_AC, DISPLAYPOLICY_DC,
+        DISPLAY_BRIGHTNESS, IOCTL_VIDEO_QUERY_DISPLAY_BRIGHTNESS,
+        IOCTL_VIDEO_QUERY_SUPPORTED_BRIGHTNESS, IOCTL_VIDEO_SET_DISPLAY_BRIGHTNESS,
+    },
+    Foundation::{CloseHandle, BOOL, HANDLE, LPARAM, PWSTR, RECT},
+    Graphics::Gdi::{
+        EnumDisplayDevicesW, EnumDisplayMonitors, GetMonitorInfoW, DISPLAY_DEVICEW, HDC, HMONITOR,
+        MONITORINFO, MONITORINFOEXW,
+    },
+    Storage::FileSystem::{
+        CreateFileW, FILE_ACCESS_FLAGS, FILE_FLAGS_AND_ATTRIBUTES, FILE_SHARE_MODE, OPEN_EXISTING,
+    },
+    System::Diagnostics::Debug::{GetLastError, ERROR_ACCESS_DENIED, ERROR_NOT_FOUND, WIN32_ERROR},
+    System::SystemServices::{DeviceIoControl, GENERIC_READ, GENERIC_WRITE},
+    UI::WindowsAndMessaging::EDD_GET_DEVICE_INTERFACE_NAME,
+};
+
+#[allow(non_snake_case)]
```

Comment:
I'd rather follow the Rust naming conventions and call the field `physical_monitor`. I don't see a good reason to deviate from them here.